analysers/experiments/cpu.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+# Takes arguments: Spark master, Cassandra host, Minio host, path of the files
+sparkMaster = sys.argv[1]
+cassandraHost = sys.argv[2]
+trialID = sys.argv[3]
+experimentID = trialID.split("_")[0]
+cassandraKeyspace = "benchflow"
+srcTable = "trial_cpu"
+destTable = "exp_cpu"
+
+# Set configuration for spark context
+conf = SparkConf() \
+    .setAppName("Cpu analyser") \
+    .setMaster(sparkMaster) \
+    .set("spark.cassandra.connection.host", cassandraHost)
+sc = CassandraSparkContext(conf=conf)
+
+# TODO: Use Spark for all computations
+
+CassandraRDD = sc.cassandraTable(cassandraKeyspace, srcTable)
+CassandraRDD.cache()
+
+def sortAndGet(field, asc):
+    v = CassandraRDD.select(field) \
+        .where("experiment_id=?", experimentID) \
+        .map(lambda x: (x[field], 0)) \
+        .sortByKey(asc, 1) \
+        .map(lambda x: x[0]) \
+        .first()
+    return v
+
+dataMin = sortAndGet("cpu_min", 1)
```

Comment:
Why can't you just sort once and get all the data you need out of the sorted RDD?